wbooks/src/app/components/CardBook/styles.js

```diff
@@ -0,0 +1,42 @@
+import { StyleSheet } from 'react-native';
+import { white } from '@constants/colors';
+
+export const styles = StyleSheet.create({
+  containerCard: {
+    display: 'flex',
+    flexDirection: 'row',
+    height: 120,
+    elevation: 2,
+    borderRadius: 5,
+    shadowOffset: {
+      width: 12,
+      height: 12
+    },
+    shadowOpacity: 16.0,
+    width: '85%',
+    marginLeft: 'auto',
+    marginRight: 'auto',
+    backgroundColor: white,
+    padding: 20,
```

Comment:
why this and the use paddingHorizontal? if you want u can do somehting like this:
``` 
paddingVertical: 20,
paddingHorizontal: 8
```